hrms/hr/doctype/leave_application/leave_application.py

```diff
@@ -786,7 +786,7 @@ def get_leave_details(employee, date):
 		}
 
 	# is used in set query
-	lwp = frappe.get_list("Leave Type", filters={"is_lwp": 1}, pluck="name")
+	lwp = frappe.get_list("Leave Type", or_filters={"is_lwp": 1,"allow_over_allocation":1}, pluck="name")
```

Comment:
This function just returns leave balance details to be displayed on the leave application dashboard. It doesn't filter out leaves with "allow over allocation" enabled.

> all leave types that have allow_over_allocation set as true should be available to the employee in leave application.

So this doesn't make sense. What is it that you are trying to fix here?

This is what the "Allow Over Allocation" setting allows:

<img width="618" alt="image" src="https://user-images.githubusercontent.com/24353136/233594301-8abb9abb-8592-41dc-a301-9b088a8cf943.png">

Reply:
Leave application has a field "leave type" that is populated from server call "get_leave_details" based on available leaves allocation to selected employee, if there is a leave type like "sick leave" which isn't allocated nor LWP , (it's paid leave). this leave type would not show in the dropdown list of leave type in leave application.